package/kernel/linux/modules/other.mk

```diff
@@ -1129,3 +1129,52 @@ endef
 $(eval $(call KernelPackage,tpm-i2c-infineon))
 
 
+define KernelPackage/gpio-ich
+  TITLE:= Intel ICH GPIO support
+  SUBMENU:=$(OTHER_MENU)
+  KCONFIG:= \
+    CONFIG_GPIO_ICH
+  FILES:= \
+    $(LINUX_DIR)/drivers/gpio/gpio-ich.ko
+  AUTOLOAD:= $(call AutoProbe, gpio-ich)
+  DEPENDS:=@GPIO_SUPPORT
+endef
```

Comment:
I meant the other way around fix gpio-ich to be like itco-wdt, so it looks like the surrounding code.